src/quo2/components/drawers/__tests__/permission_context_component_spec.cljs

```diff
@@ -0,0 +1,15 @@
+(ns quo2.components.drawers.--tests--.permission-context-component-spec
+  (:require ["@testing-library/react-native" :as rtl]
+            [quo2.components.drawers.permission-context :as permission-context]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]))
+
+(defn render-permission-context
+  ([options]
+   (rtl/render (reagent/as-element [permission-context/permission-context options]))))
```

Comment:
FYI: You can use `test-helpers.component` ns to help out with some boilerplate. For example: https://github.com/status-im/status-mobile/blob/develop/src/quo2/components/selectors/filter/component_spec.cljs#L7